src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Application {
+    private static final Random RANDOM_NUMBER = new Random();
```

Comment:
Should be inside main method
```suggestion
    private static final Random RANDOM_NUMBER = new Random();
```